package.json

```diff
@@ -81,6 +82,7 @@
     "eslint-config-prettier": "^8.6.0",
     "eslint-plugin-cypress": "^2.12.1",
     "happy-dom": "^8.1.5",
+    "jest": "^29.4.1",
```

Comment:
cc @Eakam1007, do we have an issue on switching to Jest already?  Looks like some of it is happening here, but likely it will need more work to configure it fully.